docs/querying/sql-aggregations.md

```diff
@@ -63,45 +63,46 @@ In the aggregation functions supported by Druid, only `COUNT`, `ARRAY_AGG`, and
+| `COUNT(expr, [expr...])`                                                                                      | Counts the number of rows where at least one of `expr` is not null.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |`0`|
+| `COUNT(DISTINCT expr, [expr...])`                                                                             | Counts all distinct tuples of `exprs`.E.g. For rows `(1, 2)`, `(1,1)` and `(2, 2)`, `count(distinct col1, col2)` will return 3. <br><br>When `useApproximateCountDistinct` is set to "true" (the default), this is an alias for `APPROX_COUNT_DISTINCT`. The specific algorithm depends on the value of [`druid.sql.approxCountDistinct.function`](../configuration/index.md#sql). In this mode, you can use strings, numbers, or prebuilt sketches. If counting prebuilt sketches, the prebuilt sketch type must match the selected algorithm.<br><br>When `useApproximateCountDistinct` is set to "false", the computation will be exact. In this case, each `expr` must be string or numeric, since exact counts are not possible using prebuilt sketches. In exact mode, only one distinct count per query is permitted unless `useGroupingSetForExactDistinct` is enabled.<br /><br />Counts each distinct value in a [`multi-value`](../querying/multi-value-dimensions.md)-row separately. |`0`|
```

Comment:
Some doc thoughts:

I think this piece isn't fully true anymore because `APPROX_COUNT_DISTINCT` doesn't support multiple exprs: "When `useApproximateCountDistinct` is set to "true" (the default), this is an alias for `APPROX_COUNT_DISTINCT`"

In reviewing the implementation, I believe the new syntax only works if `druid.sql.approxCountDistinct.function = APPROX_COUNT_DISTINCT_BUILTIN`. Since I don't see a mode similar to `byRow` in `APPROX_COUNT_DISTINCT_DS_HLL`, I think if you set `druid.sql.approxCountDistinct.function = APPROX_COUNT_DISTINCT_DS_HLL`, you would get some kind of error. You could add a test for this in `HllSketchSqlAggregatorTest`.

I feel it would also be clearer if the example had 4 tuples with 3 distinct, instead of 3 tuples with 3 distinct.

Putting it all together, how about something like this:

> Counts distinct values of `expr`, or distinct tuples of multiple `expr`s.
>
> When using multiple `expr`, distinct tuples are counted. For example, if you run `COUNT(DISTINCT col1, col2)` when `(col1, col2)` tuples are `(1, 2)`, `(1, 1)`, `(2, 2)`, and `(1, 1)`, then the result is 3.
>
> When `useApproximateCountDistinct` is set to "true" (the default), the computation is approximate and uses the same algorithm as `APPROX_COUNT_DISTINCT`. The specific algorithm depends on the value of [`druid.sql.approxCountDistinct.function`](../configuration/index.md#sql). In this mode, when using a single `expr`, you can use strings, numbers, or prebuilt sketches. If counting prebuilt sketches, the prebuilt sketch type must match the selected algorithm. When using multiple `expr`, you cannot include prebuilt sketches, and `druid.sql.approxCountDistinct.function` must be set to `APPROX_COUNT_DISTINCT_BUILTIN`.
>
> When `useApproximateCountDistinct` is set to "false", the computation is exact. In this case, each `expr` must be string or numeric, since exact counts are not possible using prebuilt sketches. In exact mode, only one distinct count per query is permitted unless `useGroupingSetForExactDistinct` is enabled.
>
> When using a single `expr`, each distinct value in a [`multi-value`](../querying/multi-value-dimensions.md)-column is counted separately. When using multiple `expr`, multi-value columns are counted as whole arrays.